crates/tests/Cargo.toml

```diff
@@ -1,13 +1,11 @@
 [package]
 name = "vello_tests"
+description = "Helper code for writing Vello tests."
```

Comment:
I'm not sure this is the right description here

Like yes, the lib crate only includes the helper code, but that's not the raison d'être for this package.

Reply:
I just copied this from #468's `ARCHITECTURE.md`. Also we're not publishing this crate, so it doesn't matter that much anyway.

That said, just tell me a better description and I'm all for it.

Reply:
I'd probably just have "Integration tests for Vello"